torchsynth/module.py

```diff
@@ -651,6 +648,31 @@ def _forward(self, *args: Any, **kwargs: Any) -> T:
         raise RuntimeError("TorchSynthParameters cannot be called")
 
 
+class TorchIdentity(TorchSynthModule):
```

Comment:
What is this going to be used for?

Reply:
For when you want to ignore a particular oscillator

Reply:
I was trying to replicate the old examples.py